promql/promqltest/test.go

```diff
@@ -479,43 +480,22 @@ func (cmd *loadCmd) append(a storage.Appender) error {
 	return nil
 }
 
-func getHistogramMetricBase(m labels.Labels, suffix string) (labels.Labels, uint64) {
-	mName := m.Get(labels.MetricName)
-	baseM := labels.NewBuilder(m).
-		Set(labels.MetricName, strings.TrimSuffix(mName, suffix)).
-		Del(labels.BucketLabel).
-		Labels()
-	hash := baseM.Hash()
-	return baseM, hash
-}
-
 type tempHistogramWrapper struct {
```

Comment:
Do we need to change non parser files in the same PR?

Reply:
Hmm, that's fair, I'll make a separate PR to move stuff around as prepwork.

Reply:
Thanks! https://github.com/prometheus/prometheus/pull/15156